src/interfaces/alert-i.ts

```diff
@@ -0,0 +1,10 @@
+import type { AlertTypesE } from '~/enums/alert-types-e'
+import type { AlertSeverityE } from '~/enums/alert-severity-e'
+
+export interface AlertI {
+  severity: AlertSeverityE | null
```

Comment:
are these values ever null? Or are they just undefined sometimes?

Reply:
also in general I think it is better to import enums/interfaces from '#imports' so that we don't have to update as many places if we change the location of things later (i.e. move some enum stuff to a common repo or rename the file etc.)

Reply:
Changed to undefined and to #imports